aten/src/ATen/function_wrapper.py

```diff
@@ -1331,6 +1334,7 @@ def process_redispatch_cpu_to_hb(option):
             # type: (FunctionOption) -> None
             redispatch_condition = "(c10::probe::use_hb_redispatch() && c10::probe::hb_profiler_is_in_roi() && c10::probe::hb_profiler_is_top_level() && c10::probe::should_redispatch(__PRETTY_FUNCTION__))"
```

Comment:
Here is the `redispatching` condition:
(1) Built with redispatching
(2) profiler is enabled
(3) is a top level kernel
(4) `Route` json says this kernel should be redispatched
There is also a hidden condition -- the cpu is currently running on CPU

Reply:
Oh okay, thanks for pointing it out!